app/helpers/views_helper.rb

```diff
@@ -0,0 +1,9 @@
+module ViewsHelper
+  def get_headings(kase, correspondence_type)
+    if kase.current_state == "rejected" && correspondence_type.abbreviation == CorrespondenceType.offender_sar.abbreviation
+      t4c(kase, "", t("cases.new.offender_sar.rejected"), case_type: kase.pretty_type)
```

Comment:
I think this will work:
```suggestion
      t4c(kase, "", t("rejected"), case_type: kase.pretty_type)
```